.github/CODEOWNERS

```diff
@@ -19,6 +19,11 @@
 ######################
 /packages/http-client-python/                @iscai-msft @tadelesh @msyyc @timotheeguerin @lmazuel
 
+######################
+# JavaScript
+######################
+/packages/http-client-js/                @joheredi
```

Comment:
you should add at least someone else to this because otherwise nobody can approve your prs. but as you are part of the workspace probably should also add us